apps/users/validators.py

```diff
@@ -0,0 +1,14 @@
+import re
+
+from django.core.exceptions import ValidationError
+
+
+def validate_telegram_name(value):
+    regex = re.compile(r"^[a-z][a-z0-9_]{3,30}[a-z0-9]$", flags=re.IGNORECASE)
+    regex_matches = regex.search(str(value))
```

Comment:
code-style: regex_matches - одноразовая переменная, можно сократить

Reply:
Я вижу два варианта сокращения и они мне оба не очень нравятся.

1. Получается убийственный однострочник.
```suggestion
    regex_matches = re.compile(r"^[a-z][a-z0-9_]{3,30}[a-z0-9]$", flags=re.IGNORECASE).search(str(value))
    if not regex_matches:
```

2. Не сразу понятно, во всяком случае мне, на что идет проверка, поскольку я на память не скажу, что возвращает выражение `regex.search(str(value))`.
```suggestion
    regex = re.compile(r"^[a-z][a-z0-9_]{3,30}[a-z0-9]$", flags=re.IGNORECASE)
    if not regex.search(str(value)):
```

Я все же считаю, что в данном случае создание одноразовой переменной уместно для упрощения понимания кода.

Reply:
Да первый вариант плохой. Я имел в виду второй вариант.
То, что мы кладем `regex.search(str(value))`  в переменную, не добавляет информации, что оно возвращает. 
```if not regex.search(str(value)): ``` выглядело бы хорошо